### ISSUE TITLE
Duration doesn't work

### ISSUE THREAD

**Issue by Muschasgrazias** [jenkinsci/cucumber-reports-plugin#124]
Hi,

I tried to use your plugin with an official cucumber json file:
[docs](https://www.relishapp.com/cucumber/cucumber/docs/formatters/json-output-formatter) 
one feature, one passing scenario, one failing scenario @spawn was the one I used.
But in the report the duration is always 0.

Kind regards

**Comment by damianszczepanik** [comment]
Show me JSON file

**Comment by Muschasgrazias** [comment]
```
[
  {
    "uri": "features/one_passing_one_failing.feature",
    "keyword": "Feature",
    "id": "one-passing-scenario,-one-failing-scenario",
    "name": "One passing scenario, one failing scenario",
    "line": 2,
    "description": "",
    "tags": [
      {
        "name": "@a",
        "line": 1
      }
    ],
    "elements": [
      {
        "keyword": "Scenario",
        "id": "one-passing-scenario,-one-failing-scenario;passing",
        "name": "Passing",
        "line": 5,
        "description": "",
        "tags": [
          {
            "name": "@b",
            "line": 4
          }
        ],
        "type": "scenario",
        "steps": [
          {
            "keyword": "Given ",
            "name": "this step passes",
            "line": 6,
            "match": {
              "location": "features/step_definitions/steps.rb:1"
            },
            "result": {
              "status": "passed",
              "duration": 1
            }
          }
        ]
      },
      {
        "keyword": "Scenario",
        "id": "one-passing-scenario,-one-failing-scenario;failing",
        "name": "Failing",
        "line": 9,
        "description": "",
        "tags": [
          {
            "name": "@c",
            "line": 8
          }
        ],
        "type": "scenario",
        "steps": [
          {
            "keyword": "Given ",
            "name": "this step fails",
            "line": 10,
            "match": {
              "location": "features/step_definitions/steps.rb:4"
            },
            "result": {
              "status": "failed",
              "error_message": " (RuntimeError)\n./features/step_definitions/steps.rb:4:in /^this step fails$/'\nfeatures/one_passing_one_failing.feature:10:in Given this step fails'",
              "duration": 1
            }
          }
        ]
      }
    ]
  }
]
```

**Comment by damianszczepanik** [comment]
1. This is Ruby, not JVM - this might not be fully supported
2. Duration is displayed properly but if you pass ns and expect to see ms then the value is 0ms

**Comment by vastenly** [comment]
I'm using cucumberjs + gulp-cucumber to generate json report
The structure of json report is like above with duration in ms.
A peace of json e.g.:
```
 "type": "scenario",
        "steps": [
          {
            "arguments": [],
            "keyword": "Given ",
            "line": 35,
            "name": "Item guid",
            "match": {
              "location": "features/step_definitions/smokeFlow.js:413"
            },
            "result": {
              "status": "passed"
            }
          },
          {
            "arguments": [],
            "keyword": "And ",
            "line": 36,
            "name": "Open item",
            "match": {
              "location": "features/step_definitions/smokeFlow.js:274"
            },
            "result": {
              "status": "passed",
              "duration": 67
            }
          },
          {
            "arguments": [],
            "keyword": "And ",
            "line": 37,
            "name": "Move document",
            "match": {
              "location": "features/step_definitions/smokeFlow.js:248"
            },
            "result": {
              "status": "passed",
              "duration": 1127
            }
          }
        ]
```
Cucumberjs generate duration data with ms in json file.
The plugin is showing duration data with ms.

Can somebody clarify why the plugin is not able to show/parce duration values ?
Everything is ok accept duration is alwais '0'.
I did'nt found the answer looking into the [code](https://github.com/jenkinsci/cucumber-reports-plugin)

**Comment by damianszczepanik** [comment]
The problem is that cucumber expects duration in ns https://github.com/damianszczepanik/cucumber-reporting/blob/master/src/test/resources/json/sample.json#L41 while yours look like seconds

**Comment by vastenly** [comment]
Sorry, I removed my comment, as I have found this data you linked to me. Thanks in any case.
So, your latest plugin expects nanoseconds

But the latest common library of [cucumberjs](https://github.com/cucumber/cucumber-js) is already changed from nano to milliseconds time format.
From CHANGELOG.md  ### [2.0.0-rc.0]
 * `StepResult` duration is now in milliseconds instead of nanoseconds

And json report now is generates with duration in milliseconds (not nanoseconds or seconds)

Are there any updated plugin that expects milliseconds ? Or any other solution to avoid the problem?
